src/modules/map/components/toolbox/TimeSliderDropdown.vue

```diff
@@ -0,0 +1,184 @@
+    return `${t('outside_valid_year_range')} ${firstEntry}-${lastEntry}`
+}
+
+const { tippyInstance: yearInputError, updateTippyContent } = useRangeTippy(
+    () => input.value,
+    tooltipContent
+)
+
+const displayEntry = computed({
+    get() {
+        if (isDropdownOpen.value) {
+            return inputValue.value
+        } else {
+            return props.modelValue
+        }
+    },
+    set(value) {
+        inputValue.value = value
+    },
+})
+
+const dropdownEntries = computed(() => {
+    return props.entries
+        .filter((entry) => {
+            if (inputValue.value) {
+                return entry.toString().includes(inputValue.value)
+            } else {
+                // nothing searching, return all
+                return true
+            }
+        })
+        .map((entry) => ({
+            htmlDisplay: entry.toString(),
+            emphasize: inputValue.value?.toString() || '',
+            url: entry,
+        }))
+})
+
+function toggleList() {
+    if (isDropdownOpen.value) {
+        closeList()
+    } else {
+        openList()
+    }
+}
+
+function openList() {
+    isDropdownOpen.value = true
+}
+
+function closeList() {
+    isDropdownOpen.value = false
+    inputValue.value = null
+}
+
+function chooseEntry(value) {
+    emit('update:modelValue', value)
+    yearInputError.value.hide()
+    closeList()
+}
+
+function focusSearchlist() {
+    if (inputValue.value) {
+        searchList.value.goToFirst()
+    } else {
+        searchList.value.goToSpecific(props.modelValue)
+    }
+}
+
+function onEnter() {
+    if (props.entries.includes(parseInt(inputValue.value))) {
+        chooseEntry(inputValue.value)
+        input.value.blur()
+    } else {
+        yearInputError.value.show()
+    }
+}
+
+// i18n.t isn't reactive, therefore we need to update the content
+// ourselves
+watch(
+    () => store.state.i18n.lang,
+    () => {
+        updateTippyContent(tooltipContent)
+    }
+)
+</script>
+
+<template>
+    <div v-click-outside="closeList">
+        <div class="d-flex">
+            <form
+                action=""
+                class="input-group input-group-append"
+                data-cy="searchable-dropdown"
+                @submit.prevent
+            >
+                <input
+                    ref="input"
+                    :value="displayEntry"
+                    class="form-control rounded-end-0"
+                    :class="{ 'rounded-bottom-0': isDropdownOpen }"
+                    @input="inputValue = $event.target.value"
+                    @focusin="openList"
+                    @keydown.esc.prevent="closeList"
+                    @keydown.down.prevent="focusSearchlist"
+                    @keydown.enter.prevent="onEnter"
+                />
+
+                <div class="input-group-append btn-group btn-group">
+                    <button
+                        class="btn border btn-light d-flex align-items-center rounded-start-0 rounded-end-0"
```

Comment:
For code consistency you should use `btn-outline-group` instead of `btn-light` for button groups.

Reply:
OK! The name is a bit misleading ;)